src/widget/button.rs

```diff
@@ -128,11 +136,12 @@ impl Iterator for TimesClicked {
 impl<'a> Button<'a, Image> {
 
     /// Begin building a button displaying the given `Image` on top.
-    pub fn image(image_id: widget::Id) -> Self {
+    pub fn image(image_id: widget::Id, alt_image_id: Option<widget::Id>) -> Self {
```

Comment:
Rather than adding an `Option` argument to the `image` constructor, I think this might be more idiomatic as a new builder method, so that it could be called like this:

```rust
Button::image(image_id).click_image(alt_image_id).set(id, ui);
```

Edit: on second thoughts, perhaps `.press_image` makes more sense than `.click_image`.